### PR TITLE
Add option to render evaluated images

### PR DESCRIPTION
**Pull request by hturki**
Adding an optional "render_output_path" parameter to the eval script that saves the rendered image outputs to the specified path.

### CODE REVIEW

File: nerfstudio/pipelines/base_pipeline.py [hunk 1]
```diff
@@ -173,7 +175,7 @@ def get_eval_image_metrics_and_images(self, step: int):
 
     @abstractmethod
     @profiler.time_function
-    def get_average_eval_image_metrics(self, step: Optional[int] = None):
+    def get_average_eval_image_metrics(self, step: Optional[int] = None, output_path: Optional[Path]=None):
         """Iterate over all the images in the eval dataset and get the average."""
```
Comment (tancik) [inline nerfstudio/pipelines/base_pipeline.py:179]: Add doc for `Args`

File: nerfstudio/pipelines/base_pipeline.py [hunk 2]
```diff
@@ -339,7 +341,7 @@ def get_eval_image_metrics_and_images(self, step: int):
         return metrics_dict, images_dict
 
     @profiler.time_function
-    def get_average_eval_image_metrics(self, step: Optional[int] = None):
+    def get_average_eval_image_metrics(self, step: Optional[int] = None, output_path: Optional[Path] = None):
```
Comment (tancik) [inline nerfstudio/pipelines/base_pipeline.py:344]: Add doc for `Args`

**Review by tancik — APPROVED** [review]
LGTM

**Comment by hturki** [comment]
sorry for the delay here @tancik - this is now passing the build

**Comment by smart4654154** [comment]
> Adding an optional "render_output_path" parameter to the eval script that saves the rendered image outputs to the specified path.

Thank you for your work. I encountered an error in nerfstudio (1.0.0).
I used this:ns-eval --load-config PATH --render-output-path PATH.
Do you know how to solve it? Thank you very much
image

**Pull request merged**